packages/messenger-widget/src/components/Preferences/NormalView.tsx

```diff
@@ -11,20 +11,33 @@ export function NormalView() {
         setShowPreferencesModal,
         showProfileConfigurationModal,
         setShowProfileConfigurationModal,
+        resetConfigureProfileModal,
+        preferencesOptionSelected,
+        setPreferencesOptionSelected,
     } = useContext(ModalContext);
 
-    const [optionChoosen, setOptionChoosen] = useState<any>(null);
+    // const [optionChoosen, setOptionChoosen] = useState<any>(null);
 
     /**
      *  Opens DM3 profile configuration by default if user clicked
      *  on "Configure Profile" button
      */
     useEffect(() => {
         if (showProfileConfigurationModal) {
-            setOptionChoosen(preferencesItems[1]);
+            setPreferencesOptionSelected(preferencesItems[1]);
```

Comment:
what is prefernce item[1]?

Reply:
preferences items is an array which contains all properties to be shown on UI like :- 
1. Configure profile
2. Network
3. Notification etc....

Here item[1] means by default "Configure Profile" is selected